Polyfills/XMLHttpRequest/Source/XMLHttpRequest.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <Babylon/JsRuntimeScheduler.h>
+
+#include <napi/napi.h>
+#include <UrlLib/UrlLib.h>
+
+#include <unordered_map>
+
+namespace Babylon::Polyfills::Internal
+{
+    class XMLHttpRequest final : public Napi::ObjectWrap<XMLHttpRequest>
+    {
+        static constexpr auto JS_XML_HTTP_REQUEST_CONSTRUCTOR_NAME = "XMLHttpRequest";
```

Comment:
Is `static` redundant when used with `constexpr`?

Reply:
Not semantically. I don't know exactly what the compiler would do with this, but [this](https://stackoverflow.com/questions/13865842/does-static-constexpr-variable-inside-a-function-make-sense) seems to claim it is not redundant.